.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,13 +1,11 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-title: ''
+name: Bug report about: Create a report to help us improve title: ''
 labels: ''
 assignees: ''
 
 ---
```

Comment:
That doesn't seem right?

Reply:
the main branch has diverged with huge difference but what's have been on my end and what's have been committed, still looking to the code and trying to resolve and fix issues.

Reply:
Hmm, there are not many changes to the files in `src` I hope?

I'd like to try to get this PR merged prior to the feature branches that are currently open. It would help the team to see what you changed to avoid the various issues raised by `flake` and adapt their code accordingly. (Rather than you having to fix a new round of issues.) Super helpful what you're doing!

Reply:
yes i totally agree as so many things are format related, still many possible enhancment that we can get into the code. Just working on my setup :D and pycharm hits me with few wired imports :'(